docs/making-your-own.md

```diff
@@ -135,31 +123,44 @@ see [Local Testing](/local-testing)
 - Installing SDDM the way it is doesn't work as the SDDM user needs to exist: [github issue](https://github.com/ublue-os/main/issues/224)
     - You can get around this by executing `adduser sddm` before running sddm
 
-# Best Practices
+## Best Practices
 
 The following are some best practices based on experience with learning how to move to this new model:
 
+### What repo to fork?
+
+The instructions here recommend using the [`startingpoint`](https://github.com/ublue-os/startingpoint) template because it:  
+
+- Is made to be modified and customized
+- Is more suitable for personal usage
+- Has an easier to edit configuration format, YAML, which also supports comments and inline documentation
+- Advanced customization can be accomplished at build time by [placing scripts in the appropriate folders](https://github.com/ublue-os/startingpoint/blob/template/scripts/README.md)
+
+Making a custom image by forking a differently structured repository such as [`main`](https://github.com/ublue-os/main) or [`bluefin`](https://github.com/ublue-os/bluefin) might be more appropriate if you intend on having multiple versions (such as a version with GNOME and another with KDE), but that will require you to be more knowledgeable in building containers. Modification is more involved and if you want to leverage the images already built at `ublue-os/main` or `ublue-os/nvidia` you might have to modify their `build.yml`.
+If you do decide to use [`main`](https://github.com/ublue-os/main) the *Manual setup* steps above still apply.
+
 ### You need to read the documentation
 - There's no way around this, we're terraforming a new planet, you'll save a ton of time by reading the documentation:
-    - [CoreOS](https://docs.fedoraproject.org/en-US/fedora-coreos/)
-    - [Kinoite](https://docs.fedoraproject.org/en-US/fedora-kinoite/)
     - [Silverblue](https://docs.fedoraproject.org/en-US/fedora-silverblue/)
-- Additionally, your developer users might not be familiar with [Podman](https://podman.io/)
+    - [Kinoite](https://docs.fedoraproject.org/en-US/fedora-kinoite/)
+    - [CoreOS](https://docs.fedoraproject.org/en-US/fedora-coreos/)
+    - [startingpoint](https://github.com/ublue-os/startingpoint)
+- You and your users might be more used to traditional desktops
+    - On immutable systems, most software development tools shouldn't be installed directly on your base system. They should be run separate from the host system in some way, like using [a toolbox](https://universal-blue.org/guide/toolbox/) or [Podman](https://podman.io/) directly.
```

Comment:
I like that this is brought up here, though I feel it might also be important enough to mention higher up

Reply:
I think it doesn't really even belong in the custom image guide, it's important in general.

Reply:
True. Maybe an "Introduction to Immutable Operating Systems/Images" page is warranted to fight off some of the confusion I see that some people have

Reply:
That's a really good idea.